pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -106,6 +106,8 @@ type zedagentContext struct {
 	devicePortConfigList      types.DevicePortConfigList
 	remainingTestTime         time.Duration
 	physicalIoAdapterMap      map[string]types.PhysicalIOAdapter
+	globalConfigPtr           *types.GlobalConfig
```

Comment:
This field isn't used anywhere, and I don't see how it would be usable for any of the contemplated changes.
Hence it is just a source of confusion and bugs and I suggest it be removed.

Reply:
Hmm.. Initially had it because I had errors as part of it. Now that it is in status, we don't need it anymore.. Will remove it.